packages/portfolio-contract/test/contract-setup.ts

```diff
@@ -43,16 +43,12 @@ const deploy = async (t: ExecutionContext) => {
     'axelar',
     'osmosis',
     'Polygon',
-    'optimism',
+    'Optimism',
```

Comment:
In the comment above, is "(mainnet and testnet)" out of date?

Also: I suggest changing it to a `/** ... */` style comment.